src/clj/rems/db/users.clj

```diff
@@ -123,3 +123,7 @@
       format-user
       ;; in case user attributes were not found, return at least the userid
       (assoc :userid userid)))
+
+(defn join-user [x]
```

Comment:
what's the context behind this function name? it's used in `src/clj/rems/api/services/invitation.clj:14-15` in a function called `join-dependencies`

Reply:
I added this task so we don't forget to think about this https://github.com/CSCfi/rems/issues/2714